lecture_02/assignment_01/eleni-skevaki/assignment_01.py

```diff
@@ -0,0 +1,31 @@
+"""Assignment 01: Project box to xy-plane
+"""
+from compas.artists import Artist
+from compas.datastructures import Mesh
+from compas.geometry import Box
+from compas.geometry import Frame
+from compas.geometry import Plane
+from compas.geometry import Projection
+
+# Define a Frame, which is not in the origin and a bit tilted to the world frame
+frame = Frame([1, 2, 3], [1, 0, 0], [0, 1, 1])
+
+# Create a Box with that frame
+box = Box(frame, 1, 1, 1)
+
+# Create a Projection (can be orthogonal, parallel or perspective)
+P = Projection.from_plane_and_direction(Plane.worldXY(), [-1,-1,-1])
```

Comment:
Code style: add whitespace after commas:
```suggestion
P = Projection.from_plane_and_direction(Plane.worldXY(), [-1, -1, -1])
```